rust-bot/src/main.rs

```diff
@@ -21,7 +21,7 @@ use serenity::{
 };
 
 #[group]
-#[commands(ping)]
+#[commands(ping, pin)]
```

Comment:
mostly concerned about this being correct

Reply:
This seems to work fine!